pytorch_toolkit/face_recognition/README.md

```diff
@@ -118,3 +119,89 @@ uncleaned version of the [MegaFace](http://megaface.cs.washington.edu/participat
 ## Face Recognition Demo
 
 1. For setting up demo, please go to [Face Recognition demo with OpenVINO Toolkit](./demo/README.md)
+
+## Train compressed model using [NNCF](https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/nncf)
+
+
+[NNCF](https://github.com/opencv/openvino_training_extensions/tree/develop/pytorch_toolkit/nncf) - is a framework for neural network compression using quantization and sparsification algorithms.
+
+To use NNCF as a standalone package refer to this [manual](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/nncf/docs/PackageUsage.md).
```

Comment:
Installation of nncf should be done in `init_venv.sh`, because you will get `Import error` when try to train models without `nncf`. 
Try to use `pip3 install -e ../nncf`, may be it will work.